### DISCUSSION TITLE
ignore '#!' like lua

### DISCUSSION THREAD

**Discussion by Freed-Wu** [xmake-io/xmake#5158] [Ideas]
We can use `xmake lua *.lua` to source a lua script like `lua *.lua`. However, if the first line of `*.lua` is `#!/usr/bin/env lua`, `xmake lua` will failed. real `lua` can work. How about let `xmake lua` support `#!` like real `lua`?

**Comment by waruqi** [comment]
`xmake lua` is not compatible with lua, and lua cannot execute most xmake lua scripts. You can only use xmake lua to load them.
